liquibase-extension-examples/pom.xml

```diff
@@ -100,7 +100,7 @@
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-plugin-plugin</artifactId>
-				<version>2.6</version>
```

Comment:
@nvoxland, This is a significant change in versions. Can you double-check this is safe, please? Same comment for line 147 of the liquibase-maven-plugin/pom.xml.

Reply:
Yes, this is the version of the javac compiler plugin to use in an example project. It really should be an up to date version, not sure why it's so old.

Reply:
Thanks for the explanation. Resolving this conversation.